concrete/src/Validator/ClosureValidator.php

```diff
@@ -1,56 +1,71 @@
 <?php
+
 namespace Concrete\Core\Validator;
 
+use ArrayAccess;
+use Closure;
+
+/**
+ * A generic validator cabable of describing itself and validating mixed values using closures.
+ */
 class ClosureValidator implements ValidatorInterface
 {
-    /** @var \Closure */
+    /**
+     * The closure that handles validation.
+     *
+     * @var \Closure
```

Comment:
These are pretty heavily exploded, these can be contained as defined in PSR-5 without losing readability or shortening the description: https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc-tags.md#522-var

```
/**
 * @var \Closure The anonymous function that returns requirements function(ClosureValidator $validator): array
 */
```